app/views/dealerships/edit.html.erb

```diff
@@ -3,14 +3,17 @@
   <%= form.label :name %>
   <%= form.text_field :name %>
   <br>
-  <%= form.label :is_open %>
-  <%= form.check_box :is_open, {}, true, false %>
+  <br>
+  <%= form.label :is_open? %>
+  <%= form.select :is_open, ['true', 'false'] %>
+  <br>
   <br>
   <%= form.label :max_car_capacity %>
   <%= form.number_field :max_car_capacity %>
   <br>
-  <%= form.label :is_full %>
-  <%= form.check_box :is_full, {}, true, false %>
+  <br>
+  <%= form.label :is_full? %>
+  <%= form.select :is_full, ['true', 'false'] %>
```

Comment:
⛏ Consider swapping the order of true and false here so that the default false is a little easier